### PR TITLE
Added news_flash

### PR DESCRIPTION
**Pull request by Mano3**
Scraps ynet flash news, classifies to accidents / not accidents, extracts the location of accidents and gets the geographical location. Stores all this data into a postgres DB whose dump iv'e included. 
On the main server: paths should be changed, two files should be included (the google nlp's library key and the google maps' key which I'm going to transfer) and a corn job needs to be created which includes this variables and runs the script every 5 minutes.

### CODE REVIEW

File: news_flash/.idea/dictionaries/user.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<component name="ProjectDictionaryState">
```
Comment (slallum) [inline news_flash/.idea/dictionaries/user.xml]: those files in `.idea` belong to intellij and shouldn't be in git.
put `.idea/` in a new `.gitignore` file inside `news_flash` (or make the intellij project use the parent dir as the root)

File: news_flash/db_queries.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import psycopg2
+
+
+def get_latest_date_from_db():
+    conn = psycopg2.connect("dbname=anyway_news_flash user=postgres password=123321")
```
Comment (slallum) [inline news_flash/db_queries.py]: you can use [context manager](http://initd.org/psycopg/docs/usage.html#with-statement) here so you won't have to remember to close your connections.

File: news_flash/geocode_extraction.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import googlemaps
+
+
+def geocode_extract(location, maps_key):
+    gmaps = googlemaps.Client(key=maps_key)
+    geocode_result = gmaps.geocode(location, language="iw", region="il")
+    lat = geocode_result[0]["geometry"]["location"]["lat"]
```
Comment (slallum) [inline news_flash/geocode_extraction.py]: if there are no results you'll get an error, is that OK?

File: news_flash/geocode_extraction.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    lng = geocode_result[0]["geometry"]["location"]["lng"]
+    if 29.479700 <= lat <= 33.332805 and 34.267387 <= lng <= 35.896244:
```
Comment (slallum) [inline news_flash/geocode_extraction.py]: why do you check that? are those Israel's גבולות גזרה? I'm pretty sure google maps api returns the country as well, so you can check that instead if you want to make sure. (and if not I think we need a comment here or constants that explain this)

File: news_flash/geocode_extraction.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+        return geocode_result[0]["geometry"]["location"]
+    else:
+        return {"lat": -1, "lng": -1}
```
Comment (slallum) [inline news_flash/geocode_extraction.py]: this is the edge case when there is "no result" - you can just return `None` here and check it outside (you're checking it anyway as far as I can tell)

File: news_flash/.idea/dictionaries/user.xml (hunk 1 above)
Comment (Mano3) [reply news_flash/.idea/dictionaries/user.xml]: Fixed

File: news_flash/db_queries.py (hunk 2 above)
Comment (Mano3) [reply news_flash/db_queries.py]: Fixed

File: news_flash/geocode_extraction.py (hunk 3 above)
Comment (Mano3) [reply news_flash/geocode_extraction.py]: Fixed, returns None

File: news_flash/geocode_extraction.py (hunk 4 above)
Comment (Mano3) [reply news_flash/geocode_extraction.py]: Found something and replaced this check with that. Thanks.

File: news_flash/geocode_extraction.py (hunk 5 above)
Comment (Mano3) [reply news_flash/geocode_extraction.py]: Changed. Thanks.

**Pull request merged**